actors/user/src/main/java/org/sunbird/user/actors/UserMergeActor.java

```diff
@@ -0,0 +1,128 @@
+package org.sunbird.user.actors;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.sunbird.actor.router.ActorConfig;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.HttpUtilResponse;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.*;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.models.user.User;
+import org.sunbird.telemetry.util.TelemetryUtil;
+import org.sunbird.user.service.UserService;
+import org.sunbird.user.service.impl.UserServiceImpl;
+
+import java.io.IOException;
+import java.util.*;
+
+@ActorConfig(
+        tasks = {"mergeUser"},
+        asyncTasks = {}
+)
+public class UserMergeActor extends UserBaseActor {
+    private ObjectMapper objectMapper = new ObjectMapper();
+    private UserService userService = UserServiceImpl.getInstance();
+    @Override
+    public void onReceive(Request userRequest) throws Throwable {
+        updateUserMergeDetails(userRequest);
+    }
+
+    private void updateUserMergeDetails(Request userRequest) throws IOException {
```

Comment:
Please split it into different private method.

Reply:
Now the method size is shredded.